### PR TITLE
Add an example section to the README

### PR DESCRIPTION
**Pull request by hvy**
## Motivation

Adds an example section to the README to make it easier for users to navigate to the new repository where Optuna examples are and will be placed. Note that all examples will be moved to this repository soon.

See also https://github.com/optuna/optuna/issues/2654.

## Description of the changes

See above.

### CODE REVIEW

**Comment by HideakiImamura** [comment]
@Crissman Could you review this PR?

File: README.md [hunk 1]
```diff
@@ -80,6 +80,9 @@ study = optuna.create_study()  # Create a new study.
 study.optimize(objective, n_trials=100)  # Invoke optimization of the objective function.
 ```
 
+## Examples
+
+Examples can be found in in [optuna/optuna-examples](https://github.com/optuna/optuna-examples).
```
Comment (HideakiImamura) [inline README.md:85]:
```suggestion
Examples can be found in [optuna/optuna-examples](https://github.com/optuna/optuna-examples).
```

**Review by HideakiImamura — COMMENTED** [review]
A nit comment.

File: README.md (hunk 1 above)
Comment (hvy) [reply README.md]: Thanks!

**Review by HideakiImamura — APPROVED** [review]
LGTM.

**Review by Crissman — APPROVED** [review]
LGTM

**Pull request merged**